clients/unshaded/src/main/java/tachyon/client/CacheType.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client;
+
+/**
+ * Specifies the type of data interaction with Tachyon. Reads may use either cache type.
+ */
+public enum CacheType {
+  /** Write to Tachyon */
```

Comment:
Reads can also write the Tachyon (for example read from remote/ufs and write to local). I will try to make the doc more descriptive.